radixdlt/src/main/java/com/radixdlt/consensus/VertexStore.java

```diff
@@ -92,19 +92,6 @@ public void insertVertex(Vertex vertex) throws VertexInsertionException {
 			throw new MissingParentException(vertex.getParentId());
 		}
 
-		if (vertex.getAtom() != null) {
```

Comment:
Need to add a comment here saying the current hack of automatically accepting anything in the prepare phase.

Reply:
Good point, done